packages/syft/src/syft/service/queue/zmq_queue.py

```diff
@@ -582,43 +645,43 @@ def _run(self):
                                 syft_worker_id=self.syft_worker_id,
                             )
                         except Exception as e:
-                            print(
-                                f"ERROR HANDLING MESSAGE: {e}, {traceback.format_exc()}"
-                            )
+                            logger.exception("Error while handling message. {}", e)
                         finally:
                             self.clear_job()
                     elif command == QueueMsgProtocol.W_HEARTBEAT:
                         pass
```

Comment:
NITPICK: Even here we could do `self.set_producer_alive()`, as it is a heartbeat message.

Reply:
i'll move the "self.set_producer_alive()" in this path